apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/avtale/TrygdeavtaleRequest.kt

```diff
@@ -1,10 +1,18 @@
 package no.nav.etterlatte.trygdetid.avtale
 
+import no.nav.etterlatte.libs.common.behandling.JaNei
 import java.util.UUID
 
 data class TrygdeavtaleRequest(
     val id: UUID?,
     val avtaleKode: String,
     val avtaleDatoKode: String?,
     val avtaleKriteriaKode: String?,
+    val personKrets: JaNei?,
```

Comment:
Er alt optional for en trygdeavtale?

Reply:
ikke funksjonelt, men tanken var å tillate lagring, men det er kanskje bedre med poppup advarsler eller defaultverdier kanskje

Reply:
tar det i en egen oppgave